src/bin/lychee/main.rs

```diff
@@ -219,7 +221,7 @@ fn parse_headers<T: AsRef<str>>(headers: &[T]) -> Result<HeaderMap> {
 fn parse_statuscodes<T: AsRef<str>>(accept: T) -> Result<HashSet<http::StatusCode>> {
     let mut statuscodes = HashSet::new();
     for code in accept.as_ref().split(',').into_iter() {
-        let code: reqwest::StatusCode = reqwest::StatusCode::from_bytes(code.as_bytes())?;
+        let code: http::StatusCode = http::StatusCode::from_bytes(code.as_bytes())?;
```

Comment:
Thanks for changing those from `reqwest` to `http`. Internally it is handled as a reexport in `reqwest` anyway, so using the underlying crate's structs is better. 👍